Orm/Xtensive.Orm/Orm/Logging/BaseLog.cs

```diff
@@ -61,63 +63,59 @@ public IDisposable InfoRegion(string message, params object[] parameters)
     /// <summary>
     /// Writes debug message.
     /// </summary>
-    /// <param name="message">Message to write to.</param>
-    /// <param name="parameters">Values of parameters in <paramref name="message"/>.</param>
+    /// <param name="messageId">Message to write to.</param>
+    /// <param name="parameters">Values of parameters in <paramref name="messageId"/>.</param>
```

Comment:
I missed this one last time
```suggestion
    /// <param name="messageId">Identifier of the message in resources that should be written at the begging and at the end of the region or message itself.</param>
    /// <param name="parameters">Values of parameters in <paramref name="messageId"/> (resource string or message itself).</param>
```